src/python.cpp

```diff
@@ -2424,7 +2420,11 @@ PyObjectRef r_convert_dataframe(RObject dataframe, bool convert) {
 // [[Rcpp::export]]
 PyObjectRef r_convert_date(Date date, bool convert) {
   
-  PyObject* py_date = PyDate_FromDate(date.getYear(), date.getMonth(), date.getDay());
+  PyObject* datetime = PyImport_ImportModule("datetime");
+  PyObject* py_date = PyObject_CallMethod(
+    datetime, "date", "iii", date.getYear(), date.getMonth(), date.getDay());
```

Comment:
Very pedantic nitpick: you may want to explicit cast `date.getYear()` and friends to `int`, e.g.

```
(int) date.getYear()
```

While the `getYear()` method does produce an `int`, in general, when using variadic functions, it's good practice to cast arguments to ensure they match the type expected / declared in the format string.

The code is correct as is, but I just want to call it out as a kind of best practice.

Reply:
done!